src/main/scala/GeographicConverter.scala

```diff
@@ -0,0 +1,129 @@
+import play.api.libs.json._
+
+/**
+  * Geographic Tool 1.Converter
+  *
+  * This converter has following 2 features.
+  * Feature 1. Convert latitude & longitude
+  *   Convert latitude and longitude geographic coordinates ( Japanese / Global )
+  * Feature 2. Convert Degree
+  *   Convert degree format ( DMS (139/41/30.3) / DEG (139.691749) )
+  */
+object GeographicConverter {
+  def main(args: Array[String]): Unit = {
+    //    val args = Array("j2g","35.667153", "139.779080")
+    //    val args = Array("g2j","35.670392", "139.775846")
+    //    val args = Array("dms2deg", "139/41/30.3")
+    //    val args = Array("deg2dms", "139.691749")
+    if (args.length > 2 && (args(0) == "g2j" || args(0) == "j2g")) {
+      println(convertCoordinates(args))
+    } else if (args.length > 1 && (args(0) == "dms2deg" || args(0) == "deg2dms")) {
+      println(convertDegree(args))
+    } else {
+      println("your input data is wrong.please input following parameter.")
+      println("Case1. convert latitude & longitude")
+      println("  <j2g/g2j> latitude longitude")
+      println("  'j2g' means converting latitude and longitude from Japanese geographic coordinates to Global geographic coordinates.")
+      println("Case2. convert degree")
+      println("  <dms2deg/deg2dms> degree")
+      println("  'dms2deg' means converting degree from DMS format (139/41/30.3) to DEG format (139.691749).")
+    }
+  }
+
+  /**
+    * convert Geographic Coordinates
+    *
+    * @param params
+    * @return String
+    */
+  def convertCoordinates(params: Array[String]): Map[String, Double] = {
+    var result: Array[Double] = new Array[Double](2)
```

Comment:
Thank you!
I removed at https://github.com/okoysm/geographic-tools/pull/8/commits/d9566c087110eca7cdf08f251f750587325796da .